client/components/product-variant/customer/productGridItem.js

```diff
@@ -1,22 +1,39 @@
 import React from "react";
 import { PropTypes } from "prop-types";
+import { $ } from "meteor/jquery";
 import { replaceComponent } from "@reactioncommerce/reaction-components";
+import { Router } from "/client/api";
 import ProductGridItemCore from "/imports/plugins/included/product-variant/components/customer/productGridItem";
 
 
 class ProductGridItem extends ProductGridItemCore {
   static propTypes = {
     showFeaturedLabel: PropTypes.bool,
+    similarProducts: PropTypes.bool,
     ...ProductGridItemCore.propTypes
   };
 
   static defaultProps = {
-    showFeaturedLabel: true
+    showFeaturedLabel: true,
+    similarProducts: false
   };
 
+  handleClick = (event) => {
+    event.preventDefault();
+    Router.go(this.productURL);
+    if (this.props.similarProducts) {
+      $("html,body").animate(
```

Comment:
I think adding jQuery into React is an anti-pattern. Is there no way to do this animation with one of the React libraries we already include?